File: examples/mvp-java/src/main/java/co/windly/limbosample/presentation/main/root/home/HomeModule.java
package co.windly.limbosample.presentation.main.root.home;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class HomeModule {

  //region Contribution

  @ContributesAndroidInjector
  abstract HomeFragment contibuteAndroidInjector();

  //endregion
}

File: examples/mvp-kotlin/src/main/java/co/windly/limbosample/presentation/main/root/home/HomeModule.kt
package co.windly.limbosample.presentation.main.root.home

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class HomeModule {

  //region Contribution

  @ContributesAndroidInjector
  abstract fun contributeAndroidInjector(): HomeFragment

  //endregion
}
